docs/source/running.md

```diff
@@ -156,3 +156,12 @@ Example of the command line (with optional flag to select IB device mlx5_0 port
 $ mpirun -np 2 -env UCX_NET_DEVICES=mlx5_0:1 ./executable
 ```
 
+## Running in Docker containers
+UCX can run in a container, but requires slight adjustments:
+
+* Some transports may be unsupported, depending on the runtime configuration.
+To see the available transports use `ucx_info -d`.
+
+* In order to enable shared memory transport, ptrace capability is required (`--cap-add CAP_SYS_PTRACE`). Shared memory size can be adjusted by `--shm-size=""`.
```

Comment:
1. can you pls arrange the steps as list of params to add to docker for easy copy paste?
2. what shared memory size do we recommend?
3. not that in order to use shared memory between diff containers, add something like "--ipc host"